app/views/partials/TeamNames.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.connector.model.TeamName
+@import uk.gov.hmrc.cataloguefrontend.config.Constant
+
+@this()
+
+@(
+        controllerLinkString: String,
```

Comment:
The parameter name doesn't need to include the type (that info is already available). The name doesn't make it clear what it's used for - I'd suggest `sharedTeamsLink`

Reply:
Fixed